index.html

```diff
@@ -1,12 +1,280 @@
+          <button type="button" class="btn close-btn">X</button>
+          <div class="sidebar-wrapper">
+            <div class="sidebar-header">
+              <img
+                src="assets/images/Global-ERG-Summit-Logo-2x.png"
+                width="60%"
+                alt=""
+              />
+            </div>
+    
+            <ul class="list-group">
+              <li class="list-group-item">
+                <a href="./about.html">About</a>
+              </li>
+              <li class="list-group-item">
+                <a href="">Programs</a>
+              </li>
+              <li class="list-group-item">
+                <a href="">Join</a>
+              </li>
+              <li class="list-group-item">
+                <a href="">Sponsor</a>
+              </li>
+              <li class="list-group-item">
+                <a href="">News</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <!-- Navigation -->
+        <nav class="navbar navbar-custom">
+          <div class="nav-container">
+            <button class="navbar-toggler" type="button">
+              <i class="fas fa-bars"></i>
+            </button>
+
+            <a class="navbar-brand">
+              <img src="assets/images/Global-ERG-Summit-Logo-2x.png" alt="" />
+            </a>
+            <div class="navbar-nav-container d-flex align-items-center">
+              <ul class="navbar-nav mr-auto mb-2 mb-lg-0">
+                <li class="nav-item">
+                  <a class="nav-link" href="./about.html"
+                    >About</a
+                  >
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link" href="#">Program</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link" href="#">Join</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link" href="#">Sponsor</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link" href="#">News</a>
+                </li>
+              </ul>
+
+              <div class="header-action-btn">
+                <button type="button" class="btn btn-custom">
+                  CC Campaign
+                </button>
+              </div>
+            </div>
+          </div>
+        </nav>
+      </header>
+
+      <!-- Banner Section -->
+      <section id="banner-section">
+        <div class="banner-container">
+          <h4 class="display-4 sub-title">"Musical Concert"</h4>
+          <h1 class="display-1 main-title text-uppercase">
+            <span>We are</span> <br />
+            the song
+          </h1>
+          <p class="banner-description">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. A earum, fugiat sapiente iure aspernatur porro perspiciatis debitis ipsa vitae modi natus illum ea vel, voluptatum reprehenderit hic optio!
+          </p>

```

Comment:
- [ ] Please make the background transparent so a bit of it can be seen through this text.
### Your Implementation
![image](https://user-images.githubusercontent.com/45460192/190829097-876dc2cb-402c-4356-968d-720e36d5eb99.png)
### Required
![image](https://user-images.githubusercontent.com/45460192/190829142-f87ed6a5-53fd-49b1-838c-a655c42f8faf.png)

Reply:
Can you please explain this better

Reply:
- Hello @til2to. As you can see, the background image can be seen through the text from the original design of the home page whereas yours, it does not.